### PR TITLE
Add support for React 15

### PR DESCRIPTION
**Pull request by boxfoot**
Also made some changes to the npm scripts to support cross-platform development on windows.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
+    "build": "npm run clean && cross-env NODE_ENV=build node build.js",
     "build-example": "npm run clean-example && npm run build:webpack",
-    "build:webpack": "NODE_ENV=production webpack --config webpack.config.prod.js",
+    "build:webpack": "cross-env NODE_ENV=production webpack --config webpack.config.prod.js",
     "start": "node devServer.js",
     "lint": "eslint src",
-    "test": "./node_modules/karma/bin/karma start --single-run --no-auto-watch",
-    "test:watch" : "./node_modules/karma/bin/karma start"
+    "test": "karma start --single-run --no-auto-watch",
+    "test:watch": "karma start"
```
Comment (vayser) [inline package.json]: not sure that you should use global karma module to run test

Comment (boxfoot) [inline package.json]: It's not using global.  `npm run` behavior is to automatically add `./node_modules/.bin` to the path, so this is running the locally-installed `./node_modules/.bin/karma` which symlinks `./node_modules/karma/bin/karma` but is cleaner.

Comment (vayser) [inline package.json]: thanks for the explanation, PR will be merged

Comment (boxfoot) [inline package.json]: @vayser any sense of when you'll be able to accept this PR?  Thanks!

Comment (sabov) [inline package.json]: @vayser could you be so kind to review and merge this?

**Comment by sabov** [comment]
👍

**Comment by boxfoot** [comment]
Thanks!

**Pull request merged**